doc/index.md

```diff
@@ -1,13 +1,85 @@
-# ecospheres-isomorphe
+## Table des matières
 
-## Pourquoi ISOMorphe ?
+- [Transformations disponibles](#transformations)
+- [Pourquoi ISOmorphe ?](#pourquoi)
+- [Comment fonctionne ISOmorphe ?](#comment)
+- [Comment utiliser ISOmorphe ?](#tutoriel)
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+<br/>
 
-## Comment ça marche ?
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
-
-## Documentation des transformations
+## <a name="transformations"></a>Transformations disponibles
 
 <!-- insert:transformations_docs -->
+
+<br/>
```

Comment:
Need that one too, even worse than https://github.com/ecolabdata/ecospheres-isomorphe/pull/84/files#r1828397720